src/main/webapp/app/pages/curation/collapsible/BaseCollapsible.tsx

```diff
@@ -37,6 +37,13 @@ export interface BaseCollapsibleProps {
   showLoadingSpinner?: boolean;
 }
 
+function getDataTestid(dataTestid: string, identifier: React.ReactNode) {
```

Comment:
We allow a React.ReactNode to be passed in as the collapsible title for greater flexibility. In this case, I just do a check to make sure it's a string

Reply:
Thanks for clarifying. I wonder what's the best practice to assigning `data-testid`. 

Is it better to manually generate unique test ids for each element in a list-like component.
Or, should we query for all element with `data-testid='sample-id'` (which will return a list) and use indices to get the one you're looking for? (ie in react-testing-library, there is a function `getAllByTestId`)

Reply:
I don't think we need the extra step of querying all elements with a testid and then using the index. Seems straightforward enough to just use the title. Could even use the uuid if we want, but I think that might be overkill, especially since we control the test data.